tracim/tracim/lib/webdav/sql_resources.py

```diff
@@ -981,21 +982,24 @@ def move_file(self, destpath):
                 )
                 self.content.file_extension = new_file_extension
                 self.content_api.save(self.content)
-            else:
-                workspace_api = WorkspaceApi(self.user)
-                content_api = ContentApi(self.user)
-
-                destination_workspace = self.provider.get_workspace_from_path(
-                    destpath,
-                    workspace_api,
-                )
 
-                destination_parent = self.provider.get_parent_from_path(
-                    destpath,
-                    content_api,
-                    destination_workspace,
-                )
+            # Move
+            workspace_api = WorkspaceApi(self.user)
+            content_api = ContentApi(self.user)
 
+            destination_workspace = self.provider.get_workspace_from_path(
+                destpath,
+                workspace_api,
+            )
+            destination_parent = self.provider.get_parent_from_path(
+                destpath,
+                content_api,
+                destination_workspace,
+            )
+            if destination_parent == parent and destination_workspace == workspace:  # nopep8
```

Comment:
Instead of 

```
if destination_parent == parent and destination_workspace == workspace:  # nopep8
    [...]
else:
    # do the stuff
```

it's easier to read like this:

```
if destination_parent != parent or destination_workspace != workspace:  # nopep8
    # Avoid moving the file "at the same place" if the request does not result in a real move
    [ do the stuff ]
```